spoon.rb

```diff
@@ -0,0 +1,57 @@
+STDOUT.sync = true # DO NOT REMOVE
+# Solution for https://www.codingame.com/ide/puzzle/there-is-no-spoon-episode-1
+class Spoon
+  attr_accessor :width, :height, :nodes
```

Comment:
You don't need accessors if you're not setting these either outside of the class, or by the use of `self`